static/css/components/searchbox.less

```
@@ -0,0 +1,41 @@
+/**
+ * Search button and text input
+ * [TODO: Link to documentation]
+ */
+
+.searchbox {
+  background-color: @white;
+  border: 1px solid @dark-beige;
+  border-radius: .3em;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: .2%;
+  padding-left: .5%;
+  width: 50%;
+  height: 30px;
+}
+
+.searchbox__btn-icon {
+  background: url(/static/images/search-icon.svg) center no-repeat;
+  width: 35px;
+  height: 45px;
```

```suggestion
  height: 45px;
```
This height also isn't needed as it comes from the parent.